addons/l10n_in_pos/models/pos_order.py

```diff
@@ -7,20 +7,14 @@
 class PosOrder(models.Model):
     _inherit = 'pos.order'
 
-    @api.model
-    def _get_account_move_line_group_data_type_key(self, data_type, values, options={}):
-        res = super(PosOrder, self)._get_account_move_line_group_data_type_key(data_type, values, options)
-        if data_type == 'tax' and res:
-            if self.env['account.tax'].browse(values['tax_line_id']).company_id.country_id.code == 'IN':
-                return res + (values['product_uom_id'], values['product_id'])
-        return res
-
-    def _prepare_account_move_line(self, line, partner_id, current_company, currency_id, rounding_method):
```

Comment:
Maybe better to leave it and remove in master @jva-odoo

Reply:
@jco-odoo this method is totally removed so it does not have any logic leave it here.
but okay we can leave it if it's breaking standard version policy.

Reply:
@jva-odoo With a comment to remove in master and a branch to remove it in master